setup.py

```diff
@@ -26,10 +26,6 @@
 
 
 dev_requires = [
-    "black==23.3.0",
```

Comment:
Maybe worth including `ruff` here? I guess not strictly required but it probably makes it easier to use in folks' editors